### PR TITLE
Fixed SpeechRecognition to be an EventTarget

### PR DESCRIPTION
**Pull request by phistuck**

### CODE REVIEW

File: externs/browser/w3c_speech.js [hunk 1]
```diff
@@ -72,6 +72,7 @@ HTMLInputElement.prototype.onwebkitspeechchange;
 // W3C Web Speech API implemented in Chrome M23
 /**
  * @constructor
+ * @extends {EventTarget}
```
Comment (brad4d) [inline externs/browser/w3c_speech.js]: Since we cannot make `EventTarget` a constructor at this time, perhaps you mean `@implements`?

**Comment by brad4d** [comment]
@phistuck after changing `@extends` to `@implements`, could you confirm that the maven tests pass?
Thanks.

**Comment by phistuck** [comment]
They will probably fail because I will need to add the appropriate methods first... I might get to it later this week.

**Comment by phistuck** [comment]
@brad4d - is this approved?

**Comment by brad4d** [comment]
This is stalled, because I've been too busy with other stuff to work on it.

**Comment by brad4d** [comment]
I've now imported that change and sent it for internal tests and review with @nreid260

**Comment by brad4d** [comment]
Submitted internally. Should get pushed to GitHub tomorrow.

**Pull request closed without merging**